wagtail/management/commands/purge_embeds.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+
+from wagtail.embeds.models import Embed
+
+
+class Command(BaseCommand):
+    help = "Deletes all of the Embed model objects"
+
+    def handle(self, *args, **options):
+
+        embeds = Embed.objects.all()
+        embeds_count = embeds.count()
+
+        if embeds_count:
+            embeds.delete()
+            self.stdout.write(
+                self.style.SUCCESS(f"Successfully deleted {embeds_count} embeds")
+            )
+        else:
+            self.stdout.write("No embeds deleted")

```

Comment:
Suggestion: Rather than needing to count and do an extra query, `.delete` returns how many of an object it deleted. So you can call `Embed.objects.all().delete()`, and then change the message based on its return value. Alternatively, just output "Successfully deleted 0 embeds", because as a statement it's still technically correct.